pkg/cedar/cedarldap/translated_client.go

```diff
@@ -96,6 +97,60 @@ func (c TranslatedClient) FetchUserInfo(ctx context.Context, euaID string) (*mod
 	}, nil
 }
 
+// FetchUserInfos fetches multiple users' personal details
+func (c TranslatedClient) FetchUserInfos(ctx context.Context, euaIDs []string) ([]*models2.UserInfo, error) {
+	if len(euaIDs) == 0 {
+		appcontext.ZLogger(ctx).Error("No EUA ID specified; unable to request user info from CEDAR LDAP")
+		return nil, &apperrors.InvalidParametersError{
+			FunctionName: "cedarldap.FetchUserInfo",
+		}
+	}
+
+	idsStr := strings.Join(euaIDs, ",")
+	params := operations.NewPersonIdsParams()
+	params.Ids = idsStr
+	resp, err := c.client.Operations.PersonIds(params, c.apiAuthHeader)
+	if err != nil {
+		appcontext.ZLogger(ctx).Error(fmt.Sprintf("Failed to fetch person from CEDAR LDAP with error: %v", err), zap.String("euaIDFetched", idsStr))
```

Comment:
Small possible improvements:
* Change the error message to indicate the possibility of fetching multiple people, something like "Failed to fetch person/people..."
* `zap` has a function for logging an array of strings as a field, [`Strings`](https://pkg.go.dev/go.uber.org/zap@v1.21.0#Strings). The field in this statement would probably be better as something like `zap.Strings("euaIDsFetched", zap.Strings(euaIDs))`.